static/src/javascripts/bootstraps/commercial-ophan.dcr.js

```diff
@@ -0,0 +1,40 @@
+// @flow strict
+// Monkey patch to facilitate the removal of ophan tracking from the commercial bundle sent to DCR.
+export default {
+    // $FlowFixMe
+    record: (value) => {
+        if (window && window.guardian && window.guardian.ophan && window.guardian.ophan.record)
+        {
+            window.guardian.ophan.record(value);
+        }
```

Comment:
I think you can do this if you prefer:
```suggestion
        window?.guardian?.ophan?.record?.(value);
```